Documentation/ApiOverview/Internationalization/TranslationServer/Custom.rst

```diff
@@ -11,42 +11,41 @@ With the usage of XLIFF and the freely available `Pootle <http://pootle.translat
 translation server, companies and individuals may easily set up a custom translation server
 for their extensions.
 
-There is a signal that can be caught to change the translation server URL to use. The first
-step is to register one's code for handling the signal. Such code would be placed in an
-extension's :file:`ext_localconf.php` file:
+There is an event that can be caught to change the translation server URL to use. The first
+step is to register one's listener for the event. Such code would be placed in an
+extension's :file:`services.yml` file:
 
 .. code-block:: php
 
-   $signalSlotDispatcher = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Extbase\SignalSlot\Dispatcher::class);
-   $version = (string)\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\Information\Typo3Version::class);
-   $signalSlotDispatcher->connect(
-      version_compare($version, '7.0', '<')
-         ? 'TYPO3\\CMS\\Lang\\Service\\UpdateTranslationService'
-         : 'TYPO3\\CMS\\Lang\\Service\\TranslationService',
-      'postProcessMirrorUrl',
-      'Company\\Extension\Slots\\CustomMirror',
-      'postProcessMirrorUrl'
-   );
+   services:
+     Company\Extensions\Listener\CustomMirror:
+       tags:
+         - name: event.listener
+           identifier: 'ext-extensions/customMirror'
+           method: 'postProcessMirrorUrl'
+           event: \TYPO3\CMS\Install\Service\Event\ModifyLanguagePackRemoteBaseUrlEvent
 
-The class (slot) which receives the signal (:file:`EXT:myext/Classes/Slots/CustomMirror.php`)
-could look something like:
+
+The class (listener) which receives the event
+(:file:`EXT:extensions/Classes/Listeners/CustomMirror.php`) could look something like:
 
 .. code-block:: php
 
    <?php
-   namespace Company\Extensions\Slots;
+   namespace Company\Extensions\Listener;
+   use \TYPO3\CMS\Install\Service\Event\ModifyLanguagePackRemoteBaseUrlEvent;
    class CustomMirror {
       static protected $extKey = 'myext';
 
-      public function postProcessMirrorUrl($extensionKey, &$mirrorUrl) {
-         if ($extensionKey === self::$extKey) {
+      public function postProcessMirrorUrl(ModifyLanguagePackRemoteBaseUrlEvent $evt) {
```

Comment:
As a good example we should use `$event` instead. And also add a `:void` return type.